docs/getting_started/install_ez_enterprise.md

```diff
@@ -63,7 +88,7 @@
         This can be done in the following way:
 
         ``` bash
-        COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer create-project --keep-vcs ezsystems/ezplatform-ee my-new-ee-project
+        COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer create-project --repository=https://updates.ez.no/bul/ --keep-vcs ezsystems/ezplatform-ee my-new-ee-project
```

Comment:
I haven't been able to specify two repos on the command line. JSON is possible with `--repository='{"type":"composer","url":"https://updates.ez.no/bul/"}'` but it refuses to accept multiple repos in either array or object form. Otherwise, comma separation or multiple `--reposity` doesn't work either. Any idea, @andrerom?

Other issue: If you create-project with `bul`, you still end up with `ttl` in your composer.json, since that's what we have in git. So you have to modify it afterwards, to stay on `bul`.

Reply:
> Other issue: If you create-project with bul, you still end up with ttl in your composer.json, since that's what we have in git. So you have to modify it afterwards, to stay on bul.

hmm, that's what the change above is supposed to fix, to get meta repo with bul, and hence also sub packages with bul. (ignoring commerce here, it's a different beast)

Reply:
@andrerom Nope, I get ttl. Tried clearing composer cache, no luck. Verified that satis bul repo builds ezplatform-ee with bul content, and it does. Looks like `--repository=https://updates.ez.no/bul/` isn't doing what we expect it to.

Reply:
@andrerom This is funky. If I drop the `--keep-vcs` parameter, then it installs `bul` correctly from satis. So this works: `COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":".....","password":"........."}}}' php -d memory_limit=-1 composer.phar create-project --repository=https://updates.ez.no/bul/ ezsystems/ezplatform-ee my-new-ee-project`

Does it know that the satis package doesn't have VCS info, and prefer github for that reason??? Anyway, issue solved if we go that way.

NB: `--no-install` is good for testing, makes it much faster by not installing vendors. It is also useful to do `COMPOSER_CACHE_DIR=/dev/null`

Reply:
Ok, that explains it then. `--keep-vcs` is being proposed in another blocked PR here to be dropped, so I'll keep an eye on that and put this one on hold then.

Reply:
@glye I have removed `--keep-vcs` from here, as well as removed Commerce advance as we plan to combine them into one for the release.

Ok from you on this so we can move this forward?